pkg/wsman/amt/boot/decoder.go

```diff
@@ -55,3 +60,17 @@ func (i IDERBootDevice) String() string {
 
 	return ValueNotFound
 }
+
+func DecodeWSMANError(subCode string) error {
+	if len(subCode) > 0 {
+		modifiedString := strings.Split(subCode, ":")[1]
+		switch modifiedString {
+		case "AccessDenied":
+			return errors.New("User Consent is required for this action")
```

Comment:
I was thinking this remains an access denied error and the user consent part is inferred in Console. Reason being, if we ever consider AMT users beyond an admin user, then its possible it is a permissions issue.

Reply:
Implemented a centralized AMT Error to return this in a Go way